includes/Core/Modules/Modules.php

```diff
@@ -14,13 +14,11 @@
 use Google\Site_Kit\Core\Assets\Assets;
 use Google\Site_Kit\Core\Authentication\Clients\OAuth_Client;
 use Google\Site_Kit\Core\Modules\Module_Sharing_Settings;
+use Google\Site_Kit\Core\Modules\REST_Modules_Controller;
```

Comment:
These are both no longer used here.

Reply:
Hmm, they are still used. Can you double check? 🤔

Reply:
Sorry, I misinterpreted the hint from the IDE. They are still used, but these imports aren't necessary because the classes belong to the same namespace as the file 🙂

Reply:
Nice!